docs/aws_sso.md

```diff
@@ -0,0 +1,56 @@
+If you are working with AWS, there are two things you should avoid at all costs:
+
+* Use account’s root credentials for day-to-day work
+* Provision individual users in your account and generate credentials for them
+
+AWS Single Sign On (SSO) with Athenz allows you to avoid both. Because an Athenz
+domain is directly related to an AWS account, the same authorization mechanism
+can be leveraged for SSO to the AWS console, via SAML-based integration. The
+Identity Provider service validates the user and then uses Athenz
```

Comment:
Removed reference to Verizon Media's OKTA:
"...Identity Provider service in this equation is under Verizon Media’s control, and after
validating via normal Okta mechanisms that the user is valid, then uses Athenz..."